### DISCUSSION TITLE
Is this still actively maintained?

### DISCUSSION THREAD

**Discussion by yomansk8** [dotansimha/graphql-code-generator#7957] [General] [answered]
Hello,

I hope this question wont be offending or something, that is not the intent. 🙏 

I'm just wondering if this package is still actively maintained? I've opened a bugfix PR some times ago and I don't have any news on it.

Also, I can see that the only PRs that are merged since a long time are dependabot ones (automatic merge?).

Thanks for any clue about this, and **thanks for such a good package and the work around it!**

**Comment by charlypoly** [comment, marked as answer]
Hi @yomansk8!

That's a legit and perfect timed question!

We will soon start new refactoring around the CLI and the core of GraphQL Code Generator and also work on planning the future of GraphQL Code Generator as a new major version.
As part of this project, we will start gathering feedback from the community to write and publish a public RFC as a GitHub issue.

Regarding the PR review cadence, we have been mainly focusing on [Yoga](https://graphql-yoga.com/) and [Hive](https://graphql-hive.com/) (and [other projects](https://www.the-guild.dev/open-source)) in the last few weeks, but I will take a look at the pending PRs ASAP.

Thank you very much for your support!

**Reply by yomansk8** [reply]
Hi @charlypoly !

Thank you for those information!

Can't wait to see what's the future of this package then, and thanks again for your good work!
